sql/updates/latest-dev.sql

```diff
@@ -1 +1,176 @@
+    --     (NOT is_time_interval AND time_interval IS NULL AND integer_interval IS NOT NULL)
+    -- )
+    );
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.valid_ts_interval(invl _timescaledb_catalog.ts_interval) RETURNS BOOLEAN AS
+'SELECT (invl.is_time_interval AND invl.time_interval IS NOT NULL AND invl.integer_interval IS NULL) OR (NOT invl.is_time_interval AND invl.time_interval IS NULL AND invl.integer_interval IS NOT NULL)' LANGUAGE SQL ;
+
+-- ALTER TABLE _timescaledb_config.bgw_policy_drop_chunks ADD COLUMN older_than_new _timescaledb_catalog.ts_interval;
+
+-- UPDATE _timescaledb_config.bgw_policy_drop_chunks cc
+-- SET older_than_new = ROW(TRUE,older_than,NULL)::_timescaledb_catalog.ts_interval ;
+
+-- ALTER TABLE _timescaledb_config.bgw_policy_drop_chunks ALTER COLUMN older_than_new SET NOT NULL;
+
+DROP VIEW IF EXISTS timescaledb_information.drop_chunks_policies;
+DROP VIEW IF EXISTS timescaledb_information.policy_stats;
+
+CREATE TABLE IF NOT EXISTS _timescaledb_config.bgw_policy_drop_chunks_tmp (
+    job_id          		    INTEGER                 PRIMARY KEY REFERENCES _timescaledb_config.bgw_job(id) ON DELETE CASCADE,
+    hypertable_id   		    INTEGER     UNIQUE      NOT NULL REFERENCES _timescaledb_catalog.hypertable(id) ON DELETE CASCADE,
+    older_than	    _timescaledb_catalog.ts_interval    NOT NULL,
+	cascade					    BOOLEAN                 NOT NULL,
+    cascade_to_materializations BOOLEAN                 NOT NULL,
+    CONSTRAINT valid_older_than CHECK(_timescaledb_internal.valid_ts_interval(older_than))
+);
+
+INSERT INTO _timescaledb_config.bgw_policy_drop_chunks_tmp
+(SELECT job_id, hypertable_id, ROW('t',older_than,NULL)::_timescaledb_catalog.ts_interval  as older_than, cascade, cascade_to_materializations
+FROM _timescaledb_config.bgw_policy_drop_chunks);
+
+ALTER EXTENSION timescaledb DROP TABLE _timescaledb_config.bgw_policy_drop_chunks;
+DROP TABLE _timescaledb_config.bgw_policy_drop_chunks;
+
+
+CREATE TABLE IF NOT EXISTS _timescaledb_config.bgw_policy_drop_chunks (
+    job_id          		    INTEGER                 PRIMARY KEY REFERENCES _timescaledb_config.bgw_job(id) ON DELETE CASCADE,
+    hypertable_id   		    INTEGER     UNIQUE      NOT NULL REFERENCES _timescaledb_catalog.hypertable(id) ON DELETE CASCADE,
+    older_than	    _timescaledb_catalog.ts_interval    NOT NULL,
+	cascade					    BOOLEAN                 NOT NULL,
+    cascade_to_materializations BOOLEAN                 NOT NULL,
+    CONSTRAINT valid_older_than CHECK(_timescaledb_internal.valid_ts_interval(older_than))
+);
+
+SELECT pg_catalog.pg_extension_config_dump('_timescaledb_config.bgw_policy_drop_chunks', '');
+
+INSERT INTO _timescaledb_config.bgw_policy_drop_chunks
+(SELECT * FROM _timescaledb_config.bgw_policy_drop_chunks_tmp);
+
+DROP TABLE _timescaledb_config.bgw_policy_drop_chunks_tmp;
+
+GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_catalog TO PUBLIC;
```

Comment:
hm, ok can change that. 
But looks like tables.sql grants all permissions anyway and looks like there is a good reason for it.
is there any reason it should be different in the update script ?
```
-- Set table permissions
-- We need to grant SELECT to PUBLIC for all tables even those not
-- marked as being dumped because pg_dump will try to access all
-- tables initially to detect inheritance chains and then decide
-- which objects actually need to be dumped.
GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_catalog TO PUBLIC;
GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_config TO PUBLIC;
GRANT SELECT ON ALL TABLES IN SCHEMA _timescaledb_internal TO PUBLIC;
GRANT SELECT ON ALL SEQUENCES IN SCHEMA _timescaledb_catalog TO PUBLIC;
GRANT SELECT ON ALL SEQUENCES IN SCHEMA _timescaledb_config TO PUBLIC;
GRANT SELECT ON ALL SEQUENCES IN SCHEMA _timescaledb_internal TO PUBLIC;
```